assets/src/domain/eventEditor/ui/datetimes/datesList/tableView/TableView.tsx

```diff
@@ -29,13 +29,13 @@ const TableView: React.FC<TableViewProps> = ({ className, displayStartOrEndDate,
 
 	return (
 		<ResponsiveTable
+			bodyRows={formRows}
 			className={{ tableClassName }}
 			headerRows={[headerRows]}
 			metaData={{
 				tableId: 'date-entities-list-view',
-				tableCaption: __('Event Dates', 'event_espresso'),
+				tableCaption: __('Event Datesaaas', 'event_espresso'),
```

Comment:
😲 ???

Reply:
oops

Reply:
no, Bogdan was sleeping when the change has been made